### PR TITLE
Parsing

### PR DESCRIPTION
**Pull request by Nicholas-Baron**
Code for parsing

### CODE REVIEW

File: calculator/src/tokens.kt [hunk 1]
```diff
@@ -4,10 +4,11 @@ sealed class Token
 data class Floating(val value: Float) : Token()
 data class Integer(val value: Int) : Token()
 
-//singleton
 object LeftParenthesis : Token()
 object RightParenthesis : Token()
 
+// TODO: Change the classes to objects
```
Comment (hm-lnkd) [inline calculator/src/tokens.kt:10]: perhaps adhere to the class structure presented in shunting branch or present a new standard for all branches to use

**Review by hm-lnkd — CHANGES_REQUESTED** [review]
everything looks great and the operator map was pretty intuitive i would only say to change the tokens.kt class

File: calculator/src/tokens.kt (hunk 1 above)
Comment (Nicholas-Baron) [reply calculator/src/tokens.kt:10]: I have a different branch which changes the class hierarchy of the tokens. It will be merged in the second sprint.

**Comment by hm-lnkd** [comment]
Nice everything looks great I'll approve rn

**Review by hm-lnkd — APPROVED** [review]

**Pull request merged**